src/app/productConfig/productConfigModal/productConfig.modal.component.js

```diff
@@ -21,6 +21,8 @@ class ProductConfigModalController {
   $onInit(){
     this.initModalData();
     this.initializeParams();
+
+    this.isMobile = this.$window.innerWidth <= 575;
```

Comment:
Do we do this anywhere else? Seems like it might be good to store as a constant somewhere.